src/outfit.ts

```diff
@@ -101,6 +101,30 @@ const modeableCommands = [
   "parka",
 ] as const;
 
+export type EquipResult =
+  | {
+      success: true;
+    }
+  | {
+      success: false;
+      reason: string;
+    };
+
+function fail(msg: string): EquipResult {
+  return {
+    success: false,
+    reason: msg,
+  };
+}
+
+const SUCCESS: EquipResult = { success: true };
```

Comment:
This being mutable and reused leads to an extremely, extremely funny failure case